src/handbook/product/strategy.md

```diff
@@ -4,93 +4,67 @@ navTitle: Strategy
 
 # Product Strategy
 
-Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowFuse’s vision, to become the leading "Low-Code IIoT Application Platform".
-FlowFuse will allow everyone to build custom applications to achieve their business goals and depend less on off-the-shelf software.
-It’s a draft, as everything in the handbook is a draft, so we will revisit regularly and iterate when there is a reason to do so. FlowFuse is a 100 percent agile company, so our product is; timing is everything.
-Have a look at our [roadmap](/product/roadmap/) for what we're pursuing in the upcoming releases.
+Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowFuse’s [vision and strategy](..//company/strategy.md). FlowFuse will allow everyone to build custom applications to achieve their business goals and depend less on off-the-shelf software.  It is a living strategy, so we will revisit on a regular basis and iterate when there is reason to do so; timing is everything. 
```

Comment:
```suggestion
Our strategy is an indication and statement of intent for the next 2 years, and reflects FlowFuse’s [vision and strategy](/handbook/company/strategy.md). FlowFuse will allow everyone to build custom applications to achieve their business goals and depend less on off-the-shelf point solutions. 

It strategy is ever-changing, it will be revised on a regular basis and we will iterate when there is reason to do so; timing is everything. 
```